.buildkite/scripts/make_image.sh

```diff
@@ -16,7 +16,7 @@ docker build -t helium:$DOCKER_NAME -f "${DOCKERFILE_NAME}" .
 docker tag helium:$DOCKER_NAME "$MINER_REGISTRY_NAME:$DOCKER_NAME"
 docker push "$MINER_REGISTRY_NAME:$DOCKER_NAME"
 # only update latest if github tag contains `_GA$`
-if [ "$BUILDKITE_TAG" =~ _GA$ ]; then
+if [[ "$BUILDKITE_TAG" =~ _GA$ ]]; then
```

Comment:
I had to look up this syntax. The trailing $ confused me until I saw that =~ was bash’s regex operator (so we’re looking for _GA at end of the line). 

I’ll approve this, but I almost think grep call makes sense here

Reply:
Actually, I’m even more confused now. The comment above implies that we’re searching for a literal dollar sign. Is that correct?

Reply:
I can add additional comment if you feel it would be more helpful

Reply:
Yes please. Or at least correct the comment if we are not looking for a literal '$' but using it in the regex sense for end of line